Formula/cacert.rb

```diff
@@ -0,0 +1,88 @@
+class Cacert < Formula
```

Comment:
```suggestion
class CaCertificates < Formula
```
Maybe?

Reply:
I'm fine with that. I have no strong feelings on the name.